db/default.sql

```diff
@@ -195,61 +195,64 @@ CREATE TABLE element (
   org_id          BIGINT,
   registry_id     INTEGER REFERENCES registry (ID)
     ON DELETE SET NULL
+    ON UPDATE CASCADE,
```

Comment:
move please all changes in db to separate file with name V3__......sql like this one
https://github.com/ioFog/FogController/blob/develop/db/scripts/V2__def_reg_is_private.sql

Reply:
done